src/lib/storage/chunk.hpp

```diff
@@ -52,11 +52,11 @@ class Chunk : private Noncopyable {
 
   void mark_immutable();
 
-  // Atomically replaces the current column at column_id with the passed column
-  void replace_column(size_t column_id, const std::shared_ptr<BaseColumn>& column);
+  // Atomically replaces the current segment at cxlumn_id with the passed segment
+  void replace_segment(size_t cxlumn_id, const std::shared_ptr<BaseSegment>& segment);
 
-  // returns the number of columns (cannot exceed ColumnID (uint16_t))
-  uint16_t column_count() const;
+  // returns the number of segments (cannot exceed CxlumnID (uint16_t))
+  uint16_t cxlumn_count() const;
```

Comment:
The number of columns and the number of segments are equivalent here, but I think we should keep code and comment consistent.

Reply:
Wait what? That looks confusing.

What do you propose, Arne?

Reply:
Returns the number of columns.

Reply:
Fixed.